en/docs/guides/mfa/x509.md

```diff
@@ -0,0 +1,400 @@
+        validator, to `            false           `.
+
+    Example:
+
+    ``` java
+    <?xml version="1.0" encoding="ISO-8859-1"?> <CertificateValidation xmlns="http://wso2.org/projects/carbon/certificate-validation.xml">
+     <Validators>
+     <Validator name="org.wso2.carbon.identity.x509Certificate.validation.validator.CRLValidator" displayName="CRLValidator" enable="false">
+                     <Parameter name="priority">1</Parameter>
+                     <Parameter name="fullChainValidation">true</Parameter>
+                     <Parameter name="retryCount">2</Parameter>
+        </Validator>
+        <Validator name="org.wso2.carbon.identity.x509Certificate.validation.validator.OCSPValidator" displayName="OCSPValidator" enable="false">
+                     <Parameter name="priority">2</Parameter>
+                     <Parameter name="fullChainValidation">true</Parameter>
+                     <Parameter name="retryCount">1</Parameter>
+        </Validator>
+    </Validators>
+    </CertificateValidation>
+    ```
+
+---
+
+**Disable certificate validation in an already-started WSO2 IS pack**
+
+Follow the steps below to disable certificate validation if WSO2 Identity Server was started before.
+
+TODO: dev-portal-fragment
+
+---
+
+## Configure the Authentication Endpoint
+
+1.  Open the `          deployment.toml         ` file in
+    the `          <IS_HOME>/repository/conf/         `
+    directory.
+2.  Add the following configuration to the file.
+
+    1.  `            authentication_endpoint           ` : This is the
+        URL with the port that is secured with the certificate, e.g.,
+        `                         https://localhost:8443/x509-certificate-servlet                       `
+       . Update this based on your host name.
+    2.  `            username           ` : This attribute value will be
+        taken as the authenticated user subject identifier. Update this
+        with any of the certificate attributes, e.g., CN and Email.
+    3. `name` : This attribute identifies the authenticator that is configured as the second authentication step. 
+    4. `enable`: This attribute, when set to true makes the authenticator capable of being involved in the authentication process. 
+
+    ``` toml
+    [authentication.authenticator.x509_certificate.parameters]
+    name ="x509CertificateAuthenticator"
+    enable=true
+    AuthenticationEndpoint="https://localhost:8443/x509-certificate-servlet"
+    username= "CN"
+    ```
+
+    !!! note
+        When X509 authentication is configured as the second authentication
+        step, the certificate will be validated to check whether it is
+        associated with the authenticated user in the first authentication
+        step. For that, the `           username          ` parameter will
+        be used. For that, the authenticated user name considered in the
```

Comment:
'For that' is repeated twice in two sentences back to back. Shall we rephrase this part?